WKData/Sources/WKData/Data Controllers/WKGrowthTasks/WKGrowthTasksDataController.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+public final class WKGrowthTasksDataController {
+
+    var service = WKDataEnvironment.current.mediaWikiService
```

Comment:
Maybe make this `private let` if you don't expect to use or change it outside of this class.